x-pack/solutions/security/plugins/security_solution/public/data_view_manager/hooks/use_selected_patterns.ts

```diff
@@ -7,10 +7,11 @@
 
 import { useMemo } from 'react';
 import type { DataViewManagerScopeName } from '../constants';
-import { useDataViewSpec } from './use_data_view_spec';
+import { useDataView } from './use_data_view';
 
 export const useSelectedPatterns = (scope: DataViewManagerScopeName): string[] => {
-  const { dataViewSpec } = useDataViewSpec(scope);
+  const { dataView } = useDataView(scope);
+  const indexPattern = dataView?.getIndexPattern() ?? '';
 
-  return useMemo(() => dataViewSpec?.title?.split(',') ?? [], [dataViewSpec?.title]);
+  return useMemo(() => indexPattern.split(','), [indexPattern]);
```

Comment:
Isn't the result different than before when title is an empty string or undefined?
Before we were returning an empty array `[]` and now we're returning  `['']`, unless I'm mistaken?

Also it would be good to add a unit test to verify that. Right now we only have one unit test that verifies the happy path of a value being there

Reply:
Ah yes, you are right. Missing index pattern is not possible though :D